JenkinsfileRT-sdp

```diff
@@ -0,0 +1,52 @@
+// [skip ci] and [ci skip] have no effect here.
+if (utils.scm_checkout(['skip_disable':true])) return
+
+jobconfig = new JobConfig()
+jobconfig.enable_env_publication = true
+jobconfig.publish_env_on_success_only = true
+
+// Define python version for conda
+python_version = '3.7'
```

Comment:
I believe that as of today DMS is still using 3.6.x for all subsystems.

Reply:
Right.  We will want to switch to 3.7.x eventually when we deliver B7.4 though, right?  I guess we should discuss when we should make this switchover for our own development and testing, and on which test runs.  The dailies or just the weekend runs?